changelog/august2024/2024-08-07-console-fixed-quota-consumption-is-now-displayed-at-org.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: Quota consumption is now displayed at Organization level
+status: fixed
+author:
+    fullname: 'Join the #iam channel on Slack.'
+    url: 'https://slack.scaleway.com'
+date: 2024-08-07
+category: console
+product: console
+---
+
+Quota consumption on Console is now displayed at the Organization level - and not at the Project level anymore.
+
+Since quotas are applied at the Organization level, it is now easier to follow your resource consumption and anticipate quota requests.
```

Comment:
```suggestion
Quota consumption is now displayed at the Organization level on the Console, rather than at the Project level.

Since quotas are applied at Organizatuin level, this improvement makes it easier to monitor your resource consumption and anticipate quota requests.
```